parser/parser_test.go

```diff
@@ -12,7 +12,7 @@ func TestComments(t *testing.T) {
 	let x = 5; // comment
 	// another comment
 	1 + 2
-	`;
```

Comment:
no changes to this file but for some reason go fmt doesn't like these couple lines. Might as well appease it?